src/main/java/jarvis/tasks/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package jarvis.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import jarvis.Ui;
+
+/**
+ * Represents the "Deadline" task in Jarvis app.
+ */
+public class Deadline extends Task {
+
+    private LocalDateTime dueDate;
```

Comment:
Maybe rename to `dueDateTime` to be more consistent with the other variables such as `toDateTime` and `fromDateTime`?
```suggestion
    private LocalDateTime dueDateTime;
```